en_ewt-ud-train.conllu

```diff
@@ -98954,7 +98954,7 @@
 # text = The real focus of the meeting will be on how we should build these services to meet the needs of businesses operating in B2B marketplaces.?
 1	The	the	DET	DT	Definite=Def|PronType=Art	3	det	3:det	_
 2	real	real	ADJ	JJ	Degree=Pos	3	amod	3:amod	_
-3	focus	focus	NOUN	NN	Number=Sing	13	nsubj	13:nsubj	_
+3	focus	focus	NOUN	NN	Number=Sing	13	csubj	13:csubj	_
```

Comment:
"focus" is not the head of the clause so I don't think it should be csubj

Reply:
Right...so "focus" should be `nsubj` of "be" ?

Reply:
Yeah, "be" should be made the root of the sentence (although it is a copula, under the current policy preventing double subjects).